WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -46,13 +78,112 @@ private extension ReviewsDashboardCard {
+        HStack(alignment: .top) {
+            Image(systemName: "bubble.fill")
+                .foregroundStyle(review.status == .hold ? Color.secondary : Color(.wooCommercePurple(.shade60)))
+                .padding(.horizontal, Layout.padding)
+                .padding(.vertical, Layout.cardPadding)
+
+
+            VStack(alignment: .leading) {
+                // TODO: use actual product name
+                authorText(author: review.reviewer, productName: "Fallen Angel Candelabra")
+                    .bodyStyle()
+                    .padding(.trailing, Layout.padding)
+                reviewText(text: review.review, shouldDisplayStatus: review.status == .hold)
+                    .lineLimit(2)
+                    .subheadlineStyle()
+                    .padding(.trailing, Layout.padding)
+                    .renderedIf(review.review.isNotEmpty)
+                HStack(spacing: Layout.starRatingSpacing) {
+                    ForEach(0..<abs(review.rating), id: \.self) { _ in
+                        Image(systemName: "star.fill")
+                            .resizable()
+                            .frame(width: Constants.starSize * scale, height: Constants.starSize * scale)
+                    }
+                }
+
+                Divider()
+                    .padding(.vertical, Layout.dividerSpacing)
+                    .renderedIf(!isLastItem)
+            }
+        }
+    }
+
+    func authorText(author: String, productName: String) -> some View {
+        if author.isNotEmpty {
+            return Text(String.localizedStringWithFormat(Localization.completeAuthorText, author, productName))
+        } else {
+            return Text(String.localizedStringWithFormat(Localization.incompleteAuthorText, productName))
+        }
+    }
+
+    func reviewText(text: String, shouldDisplayStatus: Bool) -> some View {
+        if shouldDisplayStatus {
+            return Text(Localization.pendingReview).foregroundColor(Color(uiColor: .wooOrange)) +
+                Text(" • " + text)
+        } else {
+            return Text(text)
+        }
+    }
+
+    var viewAllReviewsButton: some View {
+        Button {
+            /* TODO */
+        } label: {
+            HStack {
+                Text(Localization.viewAll)
+                Spacer()
+                Image(systemName: "chevron.forward")
+                    .foregroundStyle(Color(.tertiaryLabel))
+            }
+        }
+        .disabled(viewModel.syncingData)
+    }
+
 }
 
 private extension ReviewsDashboardCard {
     enum Layout {
         static let padding: CGFloat = 16
+        static let cardPadding: CGFloat = 4
         static let cornerSize = CGSize(width: 8.0, height: 8.0)
         static let hideIconVerticalPadding: CGFloat = 8
+        static let starRatingSpacing: CGFloat = 4
+        static let dividerSpacing: CGFloat = 4
+    }
+
+    enum Constants {
+        static let starSize: CGFloat = 10
```

Comment:
I'm just eyeballing the size here comparing to the design, because I'm not sure how/what's the exact size in Figma (in Figma the star is a 14px text)

Reply:
I believe we want to follow the design in `ProductReviewTableViewCell`. The star size in that view is 13pt - I haven't checked if it's still too big in this case. I'm ok with keeping 10.

Reply:
Thanks for that. I checked and saw that etting size at 13 makes it too large. I see that it is also affected by device font size, but below you can also see that even with device font smaller than Figma, the star is still too big at 13. Let's keep it 10 for now.


![Screenshot 2024-05-15 at 18 15 58](https://github.com/woocommerce/woocommerce-ios/assets/266376/9802bd51-7254-4d25-bf56-53f52bae3208)